<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_ThreadInfoGroupBox</class>
 <widget class="QWidget" name="w_ThreadInfoGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThreadInfoGroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-style: solid;
border-width: 1px;
</string>
  </property>
  <widget class="QLabel" name="l_ParentIdentifier">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[ Thread - Parent Identifier ]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_Note">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>[ Thread - Note ]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_ID">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[ Thread- ID ]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_MyParentConnector">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>[ Thread - myParentConnector ]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
